K8s-deployment/Charts/consent-validator/example-secrets/config.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "1.0",
+    "postgresOptions": {
+        "databaseIP": "database.ip",
```

Comment:
You can put the Postgres service here. And for the rest of the config as well, try to set the values which we know won't change. Like the db name, db users.

Reply:
@pranavv0 Please incorporate all config changes as well.